data/mods/More_Survival_Tools/items.json

```diff
@@ -20,8 +19,47 @@
     "turns_per_charge": 0,
     "ammo": "tinder",
     "revert_to": "null",
+    "use_action": {
+        "type": "transform",
+        "msg": "You light the tinder.",
+        "target": "hobo_stove_on",
+        "active": true,
+        "need_fire": 1,
+        "need_fire_msg": "You need a lighter or fire to light this."
+    }
+  },
+  {
+    "id": "hobo_stove_on",
```

Comment:
As for your question, I don't think needing to light it is as much a balancing factor as much as an oversight that is getting fixed now.